nats-base-client/types.ts

```diff
@@ -59,18 +59,17 @@ export enum Payload {
 }
 
 export interface ConnectionOptions {
+  authenticator?: Authenticator;
   debug?: boolean;
   headers?: boolean;
-  maxPingOut?: number;
```

Comment:
It happens very often in your PRs that fields like that (or errors) move location in that list. Is that an automatic re-ordering from your IDE or something? It is very distracting and always have to look if it has been completely removed or just moved, etc..

Reply:
sorting them - this is outstanding, so I moved it so I could track it - working in this feature atm.